recipe/meta.yaml

```diff
@@ -116,6 +116,12 @@ outputs:
         - ray.runtime_context
         - ray.state
         - ray.worker
+      commands:
+        - python -c "import ray; ray.init()"
+        # init-code seemingly depends on platform or other ambient things;
+        # the following doesn't get triggered in CI, but was a problem in
+        # https://github.com/conda-forge/ray-packages-feedstock/issues/16
+        - python -c "import ray.metrics_agent"
```

Comment:
you can move that to upper part, somewhere around `- ray.worker` on line 118 - that section handles simple `import`s just fine

Reply:
I've moved that myself, hope you won't mind me touching your PR :)

Reply:
No problem whatsoever, that's what co-maintainership is for!